lib/bamboo/plug/helper.ex

```diff
@@ -17,6 +17,12 @@ defmodule Bamboo.EmailPreviewPlug.Helper do
     |> Enum.join(", ")
   end
 
+  def format_values(values) when is_binary(values), do: values
+  def format_values(values) when is_list(values) do
+    Enum.join(values, ", ")
+  end
+  def format_values(values), do: inspect(values)
```

Comment:
I actually called it `format_headers` first, but then I realized it's not actually doing anything header specific. It just takes a string or list and formats it. So, it can be used for other stuff as well.

But sure, I can rename it if you want, just let me know and I'll fix it :)

The catch all was just because I don't actually know if there can be other values. Can it be a map or tuple? If it can't it's not needed of course :)